cloudify_cli/commands/plugins.py

```diff
@@ -100,7 +102,8 @@ def upload(ctx, plugin_path, private_resource, logger, client, tenant_name):
     if tenant_name:
         logger.info('Explicitly using tenant `{0}`'.format(tenant_name))
     parsed_url = urlparse(plugin_path)
-    if not parsed_url.scheme or not parsed_url.netloc:
+    if (not parsed_url.scheme or not parsed_url.netloc) and not \
```

Comment:
how about factoring this condition out so that it's easier to understand/self-documenting - something like `is_local = not parsed_url.scheme or not parsed_url.netloc`?